config/ember-try.js

```diff
@@ -154,6 +154,23 @@ module.exports = function() {
             }
           }
         },
+        {
+          name: 'with-ember-qunit@5.0.0-beta',
+          bower: {
+            dependencies: {
+              'ember': 'components/ember#release'
```

Comment:
why do we use bower?

Reply:
can't we use the same treatment as https://github.com/san650/ember-cli-page-object/blob/484e3fccbee26d501e3dfecfdfa110094ff6c131/config/ember-try.js#L78-L83?